tests/bootstrap.php

```diff
@@ -5,6 +5,7 @@
  * @package WP-Auth0
  */
 
+ini_set( 'error_log', '/dev/null' );
```

Comment:
Does this mean users and CI can no longer report the errors?  Or do they still go somewhere useful as well?

Reply:
`error_log` is not tested currently ... or really very useful. It's legacy functionality that still remains in a handful of API calls. We have our own error log in the plugin that reports issues. 

In short: it was not used for testing and would just output needlessly to the console.